app/src/androidTest/java/com/g11x/checklistapp/AboutActivityTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright © 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.g11x.checklistapp;
+
+import android.support.test.filters.MediumTest;
+import android.support.test.rule.ActivityTestRule;
+import android.support.test.runner.AndroidJUnit4;
+import android.widget.TextView;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.isAssignableFrom;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.core.AllOf.allOf;
+
+@MediumTest
+@RunWith(AndroidJUnit4.class)
+public class AboutActivityTest {
+  @Rule
+  public ActivityTestRule<AboutActivity> mActivity = new ActivityTestRule<>(AboutActivity.class);
```

Comment:
I haven't been using Hungarian notation since Studio highlights the members automatically (a.k.a. [His Highness](http://jakewharton.com/just-say-no-to-hungarian-notation/) said I shouldn't). I'd prefer not to use it at all, but most of all I just want to be consistent.

Thoughts?